### ISSUE TITLE
Speed up playback (2x) when long-clicking video

### ISSUE THREAD

**Issue by ThiefMaster** [voc/voctoweb#752]
This is something YouTube, TikTok, and probably many other major video sites have. When you click in the video area and hold the button down (or keep tapping in case of mobile), playback speeds up to 2x, and goes back to normal once you release it.

Especially for recordings of presentations this is great when you want to speed over less interesting areas - and much more convenient than actively changing the playback speed to something faster, and then changing it back.

**Comment by saerdnaer** [comment]
You can change the playback at lower right using following UI. We probably won't add any more features to the current player, and put existing resources into switching to a newer one which already has the feature you are suggesting, c.f. #264 .
![image](https://github.com/user-attachments/assets/0cf5d6be-59dc-40c4-960f-90269f23a529)

**Comment by ThiefMaster** [comment]
Yes, I'm aware of that UI. It's not great when you just want to skip some boring part of a video. Anyway, if you're already moving to a new player which has this, then nevermind :)

**Comment by saerdnaer** [comment]
We still have to upgrade the clappr player to its current version and provide more UI to actually be able to switch players…
